hbase1/README.md

```diff
@@ -15,17 +15,66 @@ permissions and limitations under the License. See accompanying
 LICENSE file.
 -->
 
-# HBase (1.0.x) Driver for YCSB
-This driver is a binding for the YCSB facilities to operate against a HBase 1.0.x Server cluster or Google's hosted Bigtable.
-To run against an HBase 0.98.x cluster, use the `hbase098` binding.
+# HBase (1.y) Driver for YCSB
+This driver is a binding for the YCSB facilities to operate against a HBase 1 cluster, using a shaded client that tries to avoid leaking third party libraries.
 
-See `hbase098/README.md` for a quickstart to setup HBase for load testing and common configuration details.
+# Testing HBase
+## 1. Start a HBase Server
+You need to start a single node or a cluster to point the client at. Please see [Apache HBase Reference Guide](http://hbase.apache.org/book.html) for more details and instructions.
 
-## Configuration Options
-In addition to those options available for the `hbase098` binding, the following options are available for the `hbase10` binding:
+## 2. Set up YCSB
+
+Download the [latest YCSB](https://github.com/brianfrankcooper/YCSB/releases/latest) file. Follow the instructions.
 
+## 3. Create a HBase table for testing
+
+For best results, use the pre-splitting strategy recommended in [HBASE-4163](https://issues.apache.org/jira/browse/HBASE-4163):
+
+```
+hbase(main):001:0> n_splits = 200 # HBase recommends (10 * number of regionservers)
+hbase(main):002:0> create 'usertable', 'family', {SPLITS => (1..n_splits).map {|i| "user#{1000+i*(9999-1000)/n_splits}"}}
+```
+
+*Failing to do so will cause all writes to initially target a single region server*.
+
+## 4. Run the Workload
+Before you can actually run the workload, you need to "load" the data first.
+
+You should specify a HBase config directory(or any other directory containing your hbase-site.xml) and a table name and a column family(-cp is used to set java classpath and -p is used to set various properties).
+
+```
+bin/ycsb load hbase1 -P workloads/workloada -cp /HBASE-HOME-DIR/conf -p table=usertable -p columnfamily=family
+```
+
+Then, you can run the workload:
+
+```
+bin/ycsb run hbase1 -P workloads/workloada -cp /HBASE-HOME-DIR/conf -p table=usertable -p columnfamily=family
+```
+
+Please see the general instructions in the `doc` folder if you are not sure how it all works. You can apply additional properties (as seen in the next section) like this:
+
+```
+bin/ycsb run hbase1 -P workloads/workloada -cp /HBASE-HOME-DIR/conf -p table=usertable -p columnfamily=family -p clientbuffering=true
```

Comment:
is there a way for the user to specify a specific version of the hbase-1.y client? From the poms, seems it can be specified at build time.

Reply:
you can build against a specific different version if you like yes. after this PR it's by specifying `hbase1.version` or `hbase2.version` respectively